cpp/include/cudf/aggregation.hpp

```diff
@@ -94,6 +94,7 @@ class aggregation {
    */
   enum Kind {
     SUM,              ///< sum reduction
+    SUM_ANSI,         ///< sum reduction with ANSI overflow semantics
```

Comment:
*nit*: link to "ANSI overflow semantics". An external user of this API may well have no idea what this means.